wpimath/src/main/java/edu/wpi/first/math/geometry/Rotation3d.java

```diff
@@ -400,6 +459,15 @@ public double getAngle() {
     return 2.0 * Math.atan2(norm, m_q.getW());
   }
 
+  /**
+   * Returns the angle in a measure in the axis-angle representation of this rotation.
+   *
+   * @return The angle in a measure in the axis-angle representation of this rotation.
+   */
+  public Angle getMeasure() {
```

Comment:
This should be named `getMeasureAngle()` for consistency with the other overloads.

Reply:
Originally I named it that because of Rotation2d#getMeasure()